themes/root/templates/Captcha/image.phtml

```diff
@@ -0,0 +1,8 @@
+<?php
+  $laminasCaptcha = $this->captcha->getCaptcha();
+  $id = $laminasCaptcha->generate();
+?>
+<img src="<?=$this->captcha->getCacheBasePath() . $id . $laminasCaptcha->getSuffix();?>">
+<br/><br/>
+<input name="<?=$this->captcha->getHtmlInputId()?>">
```

Comment:
Should we label this input with the same "Please enter what you see:" label as in figlet.phtml?